docs/source/en/main_classes/quantization.md

```diff
@@ -38,11 +38,21 @@ Make sure that you have installed the requirements below before running any of t
 - Latest `bitsandbytes` library
 `pip install bitsandbytes>=0.39.0`
 
-- Install latest `accelerate` from source
-`pip install git+https://github.com/huggingface/accelerate.git`
+- Install latest `accelerate`
+`pip install --upgrade accelerate`
 
 - Install latest `transformers` from source 
-`pip install git+https://github.com/huggingface/transformers.git`
+`pip install --upgrade transformers`
+
+#### Tips and best practices
+
+- **Advanced usage:** Refer to [this Google Colab notebook](https://colab.research.google.com/drive/1ge2F1QSK8Q7h0hn3YKuBCOAS0bK8E0wf) for advanced usage of 4-bit quantization with all the possible options.
+
+- **Faster inference with batch_size = 1 :** Since the `0.40.0` release of bitsandbytes, for `batch_size=1` you can benefit from fast inference. Check out [these release notes](https://github.com/TimDettmers/bitsandbytes/releases/tag/0.40.0), just make sure to have a version that is greater than `0.40.0` and you should benefit from that out of the box. 
```

Comment:
turbo nit:

```suggestion
- **Faster inference with `batch_size=1` :** Since the `0.40.0` release of bitsandbytes, for `batch_size=1` you can benefit from fast inference. Check out [these release notes](https://github.com/TimDettmers/bitsandbytes/releases/tag/0.40.0) and make sure to have a version that is greater than `0.40.0` to benefit from this feature out of the box. 
```